yogstation/code/modules/mentor/mentor.dm

```diff
@@ -104,7 +104,7 @@ GLOBAL_PROTECT(mentor_href_token)
 /proc/send_mentor_sound(client/C)
 	var/sound/pingsound = sound('yogstation/sound/misc/bikehorn_alert.ogg')
 	pingsound.volume = 90
-	if(GLOB.mentornoot || prob(10))
+	if((C.prefs.toggles & SOUND_ALT) && (GLOB.mentornoot || prob(1)))
```

Comment:
```suggestion
	if((C.prefs.toggles & SOUND_ALT) && (GLOB.mentornoot || prob(10)))
```

Reply:
why? it was 1 before

Reply:
Except it wasn't, check the changes on the PR

Reply:
oh whoops, the other one was `prob(1)`, you're right